src/systems/a320_systems/src/hydraulic.rs

```diff
@@ -842,21 +843,10 @@ impl SimulationElement for A320HydraulicLogic {
     }
 
     fn write(&self, writer: &mut SimulatorWriter) {
-        writer.write_bool("HYD_GREEN_EDPUMP_LOW_PRESS", self.green_edp_has_fault);
-
-        writer.write_bool("HYD_BLUE_EPUMP_LOW_PRESS", self.blue_epump_has_fault);
-
-        writer.write_bool("HYD_YELLOW_EDPUMP_LOW_PRESS", self.yellow_edp_has_fault);
-
-        writer.write_bool("HYD_YELLOW_EPUMP_LOW_PRESS", self.yellow_epump_has_fault);
-
-        writer.write_bool("OVHD_HYD_ENG_1_PUMP_PB_HAS_FAULT", self.green_edp_has_fault);
-        writer.write_bool(
-            "OVHD_HYD_ENG_2_PUMP_PB_HAS_FAULT",
-            self.yellow_edp_has_fault,
-        );
-        writer.write_bool("OVHD_HYD_EPUMPB_PB_HAS_FAULT", self.blue_epump_has_fault);
-        writer.write_bool("OVHD_HYD_EPUMPY_PB_HAS_FAULT", self.yellow_epump_has_fault);
```

Comment:
Writing the `..._PB_HAS_FAULT` variables here was an error: this is already handled in the `A320HydraulicOverheadPanel` type.

Reply:
Wooops :D   I knew it did but just forgot removing this :(   Better now!